src/main/java/frc/robot/commands/CMD_AutoIntake.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
```

Comment:
where did this command come from?

Reply:
It came from the branch being based from the IntakeAuto instead of main. All changes from IntakeAuto have been deleted now